book/07-extending-learners.qmd

```diff
@@ -421,14 +515,13 @@ If it runs without erroring, that's a very good start!
 
 To ensure that your learner is able to handle all kinds of different properties and feature types, we have written an "autotest" that checks the learner for different combinations of such.
 
-The "autotest" setup is generated automatically by `r ref("create_learner()")` and will open after running the function, it will have a name with the form `test_package_type_key.R`, in our case this will actually be `test_rpart_classif_rpart.key`. **This name must not be changed** as triggering automated tests rely on a strict naming scheme. In our example this will create the following script, for which no changes are required to pass (assuming the learner was correctly created):
-
-```{r 07-extending-learners-011, eval = FALSE}
-install_learners("classif.rpart")
+The "autotest" setup is generated automatically by `r ref("create_learner()")` and will open after running the function, it will have a name with the form `test_package_type_key.R`, in our case this will actually be `test_rpart_classif_rpart.R`. In our example this will create the following script, for which no changes are required to pass (assuming the learner was correctly created):
```

Comment:
```suggestion
The "autotest" setup is generated automatically by `r ref("create_learner()")` and will open after running the function.
It will have a name with the form `test_<package>_<type>_<key>.R`, in this example `test_rpart_classif_rpart.R`.
This will create the following script, for which no modifications are required (assuming the learner was correctly created):
```